app/controllers/submit.js

```diff
@@ -175,18 +169,20 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
     }),
 
     isAdmin: Ember.computed('node', function() {
+        // True if the current user has admin permissions
         // FIXME: Workaround for isAdmin variable not making sense until a node has been loaded
         let userPermissions = this.get('node.currentUserPermissions') || [];
         return userPermissions.indexOf(permissions.ADMIN) >= 0;
     }),
 
     canEdit: Ember.computed('isAdmin', 'node', function() {
+        // True if the current user is and admin and the node is not a registration.
         return this.get('isAdmin') && !(this.get('node.registration'));
```

Comment:
We are not loading registrations - this is a call to `users/user_id/nodes/` - I agree that it is good to check here too. Just in case.